openshift/kustomize/api/base/statefulset.yaml

```diff
@@ -6,6 +6,18 @@ metadata:
   namespace: default
   annotations:
     description: Deploy API
+    image.openshift.io/triggers: |-
```

Comment:
This was supposed to automate the deployment of a new image, but it doesn't work.